### ISSUE TITLE
GL2/GSp4 subgroup knowls need knowls 

### ISSUE THREAD

**Issue by edgarcosta** [LMFDB/lmfdb#5039]
The invariants displayed in the  GL2/GSp4 subgroups knowls should be knowls.

E.g.: 
https://beta.lmfdb.org/knowledge/show/gsp4.subgroup_data?label=2.6.1
https://beta.lmfdb.org/knowledge/show/gl2.subgroup_data?label=25.120.0.3

**Comment by alozanoroble** [comment]
Just to clarify, are you asking for "level", say, to be a clickable knowl? That would be useful, yes.

**Comment by AndrewVSutherland** [comment]
Yes, (and not just level, all the invariants).  This is easy to implement, I'll do it as soon as I get a chance.
